skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -101,4 +111,33 @@ abstract class AbstractSkaffoldRunConfiguration(
 
         XmlSerializer.serializeInto(this, element)
     }
+    override fun checkConfiguration(){
+        checkConfiguration(project)
+
+    }
+
+
+    companion object {
+        var skaffoldPath: Path? = Paths.get("skaffold")
```

Comment:
We probably don't need this at all - at the end we call `SkaffoldExecutorService.instance.createProcess` and if it works, it should work for general case as well.